crates/core/src/config/mod.rs

```diff
@@ -242,7 +242,7 @@ impl std::fmt::Display for Language {
                 Language::Russian => "Pусский",
                 Language::Spanish => "Español",
                 Language::Hungarian => "Magyar",
-                Language::Norwegian => "Bokmål",
+                Language::Norwegian => "Norwegian bokmål",
```

Comment:
This is supposed to be in local language so maybe "Norsk bokmål" is better?

Reply:
Agree! 
Will you  update, or should I? I'm a Git noob :)